14-minimal/README.md

```diff
@@ -0,0 +1,315 @@
+NodeJS 14 minimal container image
+=========================
+
+This container image includes Node.JS 14 on top of a minimal base image for your Node.JS 14 applications. This image is best used
```

Comment:
```suggestion
This container image includes Node.JS 14 on top of a minimal base image for your Node.JS 14 applications. This image is the best used
```

Reply:
As far as I can tell this change would make the meaning slighlty different from what I was aiming for (the image is designed to be used with the full-sized image). After the change it would mean that this image is the best image to use with the fullsized image.
Maybe just change it to "This image is designed to be used with ..."? WDYT?